### PR TITLE
add helm namespace input to helm-deploy-eks.yml

### PR DESCRIPTION
**Pull request by Kunamatata**
## Description

As part of adding `etcd` to our cluster. We need to be able to specify namespaces when running `helm upgrade --install`
This is backwards compatible since the default still defaults to `"default"`

### CODE REVIEW

**Comment by Kunamatata** [comment]
I will open a PR from main -> `v5` after this is merged into `main`

**Review by zdbrandt — APPROVED** [review]
Looks good to me but please wait for an approval from someone who has worked more with this repo.

**Review by ehaynes99 — APPROVED** [review]
Looks fine to me. `--namespace default` should be the same as not specifying it.

**Pull request merged**
